addon/lib/helperworker.js

```diff
@@ -15,9 +15,12 @@ var notifications = require("sdk/notifications");
 var { XMLHttpRequest } = require("sdk/net/xhr");
 const { watchFunction, watchWorker } = require("./errors");
 const user = require("./user");
+const {Cu} = require("chrome");
 
 var existing;
 
+Cu.import("resource:///modules/UITour.jsm");
```

Comment:
There's also the two-argument form of Cu, I think like `let UITour = {}; Cu.import("...", UITour); {UITour} = UITour;` – a little awkward, but explicit.